plugin/tokudb-backup-plugin/tokudb_backup.cc

```diff
@@ -3,6 +3,7 @@
 #ident "Copyright (c) 2014 Tokutek Inc.  All rights reserved."
 
 #define MYSQL_SERVER
+#define HAVE_REPLICATION
```

Comment:
Some necessary types and functions are not accessible without this define. I could try to pass this define from server cmake configuration. But in this case the most of the changes will be wrapped with ifdef.